### DISCUSSION TITLE
is it working under angular x?

### DISCUSSION THREAD

**Discussion by thiswallz** [fomantic/Fomantic-UI#989] [General] [answered]
Hi All,
I would like to use it with angular x.

thanks!

**Comment by y0hami** [comment]
@thiswallz Could you please specify what you mean exactly by "Angular X"

**Comment by huansern** [comment]
@hammy2899 I think he meant to say [Angular 8](https://github.com/Semantic-Org/Semantic-UI/issues/6863), which is also a topic I'm interested in. But I'm more concerned about which elements wouldn't work without JavaScript.

I assume only the elements fall under modules category in the [docs](https://fomantic-ui.com/) require JavaScript to work, is it correct?

**Comment by y0hami** [comment, marked as answer]
@huansern 
> @thiswallz I think he meant to say [Angular 8](https://github.com/Semantic-Org/Semantic-UI/issues/6863), which is also a topic I'm interested in.

If that is the case there is a few options, there is a semantic-ui angular package https://github.com/edcarroll/ng2-semantic-ui but it seems that is also inactive.
There is also this package which is specific for FUI https://www.npmjs.com/package/ngx-fomantic-ui but it seems to be a project template not a library.

Unfortunately we don't offer an official implementation because we don't have time or people to maintain it just like we don't have our own react or vue libraries.

**Comment by huansern** [comment]
@hammy2899 Is it possible to have a pure CSS/LESS only build for the upcoming v3.0 of FUI? I think by creating a CSS only build will allow FUI to be used in pretty much any project, leaving the programmatic behaviors to be implemented by end user or library author downstream.

Then there will be no more "Can I use FUI in Angular/React/Vue project? etc" kind of question. That way you don't have to spend time maintaining such libraries too.

**Comment by y0hami** [comment]
@huansern Yes the CSS only dist will most likely stay a thing however v3 is going to be SASS so there won't be a LESS dist but we are making the framework more modular so you will also be able to import specific components and keep theme support.

**Comment by huansern** [comment]
Okay thanks. You can close this issue now I suppose, since all questions already been answered and there is nothing left to discuss.
